README.md

```diff
@@ -64,7 +64,7 @@ impl Buildpack for HelloWorldBuildpack {
     // writing CNB TOML files are handled by libcnb.rs itself.
     type Error = GenericError;
 
-    // This method will be called when the CNB lifecycle calls detect. Use the DetectContext to 
+    // This method will be called when the CNB lifecycle calls `bin/detect`. Use the DetectContext to 
```

Comment:
I agree that this is inconsistent with the docs for `build`, IMO this should be `executes the detection phase`.